stubs/test.blade.php

```diff
@@ -1,6 +1,8 @@
 namespace App\Tests;
 
+@if(version_compare(app()->version(), '7', '<'))
```

Comment:
let's define local variable and use it in all cases

```php
@php($shouldUseStatusConsts = version_compare(app()->version(), '7', '<'))
namespace App\Tests;

@if($shouldUseStatusConsts)
```

Reply:
@DenTray fixed!